ui/src/app/modules/login/login.component.ts

```diff
@@ -10,6 +10,7 @@ export class LoginComponent {
   private loginError :string
 
   constructor(private router: Router, private route: ActivatedRoute) {
+    localStorage.removeItem('id_token');
```

Comment:
Will this force you to login everytime you visit the app? IE login, go to another site in the same tab/window, then go back and have to login again?